packages/wrangler/src/pages/dev.ts

```diff
@@ -42,6 +41,7 @@ import type {
 } from "../yargs-types";
 import type { RoutesJSONSpec } from "./functions/routes-transformation";
 
+export const DEFAULT_PAGES_LOCAL_PORT = 8788;
```

Comment:
can we please add this on line `78` instead 🙏, just before 👇  ? That way we'll have all these related vars grouped together

```
const DEFAULT_IP = process.platform === "win32" ? "127.0.0.1" : "localhost";
const DEFAULT_SCRIPT_PATH = "_worker.js";
```

Reply:
Sure thing, thanks @CarmenPopoviciu. Done!

Reply:
thank you so much!